makefile

```diff
@@ -13,7 +13,7 @@
 #
 
 ISVCS_NAME := serviced-isvcs
-ISVCS_VERSION     := 42-dev
+ISVCS_VERSION     := 43-dev
```

Comment:
This isn't necessary because we've never published a `42-dev`. The last published version was 41.  We need to doc a procedure for this. I think the procedure should be
* build and test locally with the latest `NN-dev` version (including testing a local CC build)
* change this makefile to drop the `-dev` suffix and open a PR and get a code review
* after the PR has been merged, build and publish the image without the `-dev` prefix (e.g. 42 in this case)
* change the makefile to use `<NN+1>-dev`
* open a CC PR to use the recently published NN version (42 in this case)